Makefile.am

```diff
@@ -3692,7 +3692,7 @@ test_krb5_wait_queue_LDADD = \
 
 test_cert_utils_SOURCES = \
     src/tests/cmocka/test_cert_utils.c \
-    src/util/cert/cert_common_p11_child.c \
+    src/responder/ssh/ssh_cert_to_ssh_key.c \
```

Comment:
Wouldn't it be better to move sssd_ssh specific test out of test_cert_utils as well?